src/dist/replication/lib/bulk_load/replica_bulk_loader.cpp

```diff
@@ -274,8 +274,50 @@ error_code replica_bulk_loader::do_bulk_load(const std::string &app_name,
 error_code replica_bulk_loader::validate_bulk_load_status(bulk_load_status::type meta_status,
```

Comment:
This function seems pure static. You should declare it static to make it clear.

```suggestion
/*static*/ error_code replica_bulk_loader::validate_bulk_load_status(bulk_load_status::type meta_status,
```